cobigen-templates/templates-devon4j/src/test/java/com/devonfw/cobigen/templates/devon4j/test/templates/SQLTemplateGenerationTest.java

```diff
@@ -0,0 +1,75 @@
+package com.devonfw.cobigen.templates.devon4j.test.templates;
+
+import static org.assertj.core.api.Assertions.*;
+
+import com.devonfw.cobigen.templates.devon4j.test.templates.testclasses.SQLTestJoinTableEntity;
+import org.junit.Test;
+
+import com.devonfw.cobigen.templates.devon4j.test.templates.testclasses.SQLTestEntity;
+import com.devonfw.cobigen.templates.devon4j.test.templates.testclasses.SQLTestDataTypesEntity;
+import com.devonfw.cobigen.templates.devon4j.test.templates.testclasses.SQLTestForeignKeysEntity;
+import com.devonfw.cobigen.templates.devon4j.utils.SQLUtil;
+
+/**
+ * Test class for SQL template generation
+ *
+ */
+public class SQLTemplateGenerationTest extends AbstractJavaTemplateTest {
+
+  @Override
+  public Class<?>[] getUtils() {
+
+    return new Class<?>[] { SQLUtil.class };
+  }
+
+  @Override
+  public String getTemplatePath() {
+
+    return "src/main/templates/sql_java_app/templates/V0000__Create_${variables.entityName}Entity.sql.ftl";
```

Comment:
Shouldn't this be moved to a constant instead of using a getter?

Reply:
The getter of the abstract class needs to be overridden for it to function.